packages/cursorless-engine/src/actions/InsertSnippet.ts

```diff
@@ -121,12 +109,12 @@ export default class InsertSnippet implements Action {
   }
 
   async run(
-    [targets]: [Target[]],
+    targets: Target[],
```

Comment:
I think this has to be a destination, because otherwise how would "snip funk after bat" work? But that should come out of the tests